Pickers/SWActionSheet.m

```diff
@@ -9,7 +9,7 @@
 
 static const float duration = .25f;
 
-static const enum UIViewAnimationOptions options = UIViewAnimationOptionCurveEaseIn;
+static const enum UIViewAnimationOptions options = 7 << 16;
```

Comment:
When you add observer to `UIKeyboardWillShowNotification` or `UIKeyboardWillChangeFrameNotification`, you can get a `UIViewAnimationCurve` value by 
```objective-c
[notification.userInfo[UIKeyboardAnimationCurveUserInfoKey] integerValue]
```

When I print the value, it give me a `7`. But there is no UIViewAnimationCurve with a rawValue 7.
Even in swift, you have to force cast it with unsafeBitCase() function.
I use `7` in:
```objective-c
    [UIView beginAnimations:nil context:nil];
    [UIView setAnimationDuration:[notification.userInfo[UIKeyboardAnimationCurveUserInfoKey] doubleValue]];
    [UIView setAnimationCurve:(UIViewAnimationCurve)[notification.userInfo[UIKeyboardAnimationCurveUserInfoKey] integerValue]];
    _textField.bottom = [notification.userInfo[UIKeyboardFrameEndUserInfoKey] CGRectValue].origin.y;
    [UIView commitAnimations];
```
It works!

But there is difference between UIViewAnimationCurve and UIViewAnimationOptions

```objective-c
    UIViewAnimationOptionCurveEaseInOut            = 0 << 16, // default
    UIViewAnimationOptionCurveEaseIn               = 1 << 16,
    UIViewAnimationOptionCurveEaseOut              = 2 << 16,
    UIViewAnimationOptionCurveLinear               = 3 << 16,
```

and this is `UIViewAnimationCurve`

```objective-c
typedef NS_ENUM(NSInteger, UIViewAnimationCurve) {
    UIViewAnimationCurveEaseInOut,         // slow at beginning and end
    UIViewAnimationCurveEaseIn,            // slow at beginning
    UIViewAnimationCurveEaseOut,           // slow at end
    UIViewAnimationCurveLinear
};
```

As a result: 
UIViewAnimationOptionCurveEaseInOut = UIViewAnimationCurveEaseInOut << 16,
UIViewAnimationOptionCurveEaseIn = UIViewAnimationCurveEaseIn << 16
.....
and so on.

So I guess, if I use UIViewAnimationOptions, I can use 7 << 16.
It turns out it totally worked!